common/test/acceptance/pages/lms/problem.py

```diff
@@ -189,23 +189,23 @@ def wait_success_notification_visible(self):
         Check for visibility of the success notification and icon.
         """
         msg = "Wait for success notification to be visible"
-        self.wait_for_element_visibility('.notification.success', msg)
```

Comment:
This actually should still work.  I am going to take another look at this before merging.